misc/test_convert.ps1

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: 2024 Carl Zeiss Microscopy GmbH
+#
+# SPDX-License-Identifier: MIT
+
+# Define the path to the executable
+$executablePath = "D:\dev\MyGitHub\ptahmose_imgdoc2\out\build\x64-Debug\convczi\convczi.exe"
```

Comment:
Not a huge problem, but may want to remove hardcoded paths here and use arguments or values set by a .env file for transient, user-specific, environment variables ([DotEnv](https://github.com/stopthatastronaut/poshdotenv) for example is a module that implements the logic for this)

Reply:
I added a comment stating that this is work-in-progress and not meant to be used as-is.